README.md

```diff
@@ -8,6 +8,9 @@ A timer component for React that counts down to zero for a specified number of m
  `initialTimeRemaining: Number`
  The time remaining for the countdown (in ms).
 
+`setClass: String (optional -- default: 'timer')`
```

Comment:
I thought about this, but didn't think it was explicit enough for what is was actually doing, if you want to always have "timer" as a class, I would recommend changing this to something like "appendClassname"